README.md

```diff
@@ -80,7 +80,7 @@ python pipx.pyz ensurepath
 
 ### Use with pre-commit
 
-pipx [has pre-commit support](docs/installation.md#pre-commit).
+pipx [has pre-commit support](installation.md#pre-commit).
```

Comment:
This means you can't use the link on GitHub, but I don't see another possibility to view it in the built documentation.

Reply:
Maybe we could add a symlink?

Reply:
You mean, link `docs/installation.md` into the project's root directory?

Reply:
Yeah